javascript/code-challenges/challenges-07.test.js

```diff
@@ -0,0 +1,359 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function called sortStarWarsCharacters that sorts the characters in the starWarsPeople array by height from tallest to shortest.
+------------------------------------------------------------------------------------------------ */
+
+let starWarsPeople = [
+  {
+    "name": "C-3PO",
+    "height": "167",
+    "eye_color": "yellow"
+  },
+  {
+    "name": "Luke Skywalker",
+    "height": "172",
+    "eye_color": "blue"
+  },
+  {
+    "name": "R2-D2",
+    "height": "96",
+    "eye_color": "red"
+  }
+];
+
+const sortStarWarsCharacters = (starWarsArr) => {
+  let newArr = starWarsArr.sort((firstName, secondName) => {
+    return  secondName.height - firstName.height;
+  });
+  return newArr;// Solution code here...
+}
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named removeThree that takes an index and an array. The function should removes three items in the array starting with the value at the index. 
+------------------------------------------------------------------------------------------------ */
+
+const removeThree = (idx, arr) => {
+ let newArr =  // Solution code here...
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function named joinArray that takes an array and joins all of the elements together in one string on a space.
+------------------------------------------------------------------------------------------------ */
+
+const joinArray = (arr) => {
+  // Solution code here...
+};
+
```

Comment:
It is easy to overthink this one, but it is fairly straightforward :) What method helps combine arr to string?